### DISCUSSION TITLE
I have an API for my App and it displays me in localhost:1234 Unhadled Rejection (Failed to fetch)

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#1554] [Help]
### Summary

I would like help to see what can I do to check wether CORS is blocking still my App or there is something else. I commented out the cors in my code to test it. Anyhow as you may see in the picture that is what I get.
I have not really done anything to my vercel.json file.

![2023-02-16 (1)](https://user-images.githubusercontent.com/91563884/219486467-e0da71d5-1a3e-4791-a9eb-28a07e25b007.png)

### Additional information

```
//Import <cors> - Middleware for controlling which domains have access
const cors = require ("cors");
let allowedOrigins = [
  "http://localhost:8080",
  "http://localhost:1234",
  "http://localhost:4200",
  "https://movies-couch-api-git-main-herra17.vercel.app/",
  "https://movies-couch-api-herra17.vercel.app/",
  "https://movies-couch-api.vercel.app/movies"
];
// check if the domain where the request came from is allowed
app.use(
  cors({
    origin: (origin, callback) => {
      if(!origin) return callback(null, true);
      if (allowedOrigins.indexOf(origin === -1)) {
        let message =
        "The CORS policy for this application doesn't allow access from origin" + origin;
        return callback(new Error(message), false); 
      }
      return callback(null, true);
    },
  }));
```
This is in my API file.
And I get the error vinculated to my client side:
under the following lines-

```
useEffect(() => {
    fetch("https://movies-couch-api.vercel.app/movies")
    .then((response) => response.json())
    .then((data) => {
        console.log(data); 
```

**Comment by tanvir1017** [comment]
I'm also getting this issue(bug). Using cors and calling it also not working

**Reply by ghost** [reply]
I think I had a typo. I do not recall the way I solved it.
